### ISSUE TITLE
Hardcoded "/usr/local" prefix in src/xjackfreak1.c

### ISSUE THREAD

**Issue by rekado** [johnhldavis/xjackfreak#3]
`src/xjackfreak1.c` contains various lines like this:

```
load_skin("/usr/local/etc/xjackfreak/led-grn-01.ppm");
```

"/usr/local" really should be replaced with the configure-time `prefix`.  Or even better: "/usr/local/etc/xjackfreak" should be replaced with the `datarootdir`.

**Comment by johnhldavis** [comment]
Hi rekado,
 thanks for the feedback.

This issue was noted and fixed a while ago. Ended up with a define pragma in xjackfreak1.c
#define &#95;&#95;PREFIX&#95;&#95; "/usr/local"
which can be set using:
./configure --prefix=/my/prefix/

If you clone/pull a recent copy, it should be there.
regards,
John
PS just noticed the v1.0 tag was a bit old: have updated it to point to the latest version (shouldn't do it but v1.1 should be ready soon :-)

**Comment by rekado** [comment]
> PS just noticed the v1.0 tag was a bit old: have updated it to point to the latest version (shouldn't do it but v1.1 should be ready soon :-)

Yeah, I just noticed this.  Please do not do this in the future, as for package managers that check the integrity of the sources against a known hash (like GNU Guix) this will result in a build failure.  I'll update the hash in the Guix package recipe now.

Thanks!

**Comment by rekado** [comment]
There's another hardcoded path in `xjackfreak-1.0/doc/Makefile.am`:

```
docdir= /usr/share/doc/${PACKAGE}-${PACKAGE_VERSION}
```

It would be better if this was using the prefix instead of `/usr`.

**Comment by johnhldavis** [comment]
Well spotted.
I've updated it but not entirely sure it fully complies with Auto-tools/GNU Coding standards.
